engine/src/flutter/shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -931,6 +931,13 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
     if (Build.VERSION.SDK_INT >= API_LEVELS.API_28) {
       if (semanticsNode.tooltip != null) {
         result.setTooltipText(semanticsNode.tooltip);
+        // Tooltips are not announced when a node is focused resulting in no
+        // message. This is only announced after a long press and the tooltip.
+        // To be consistent with other platforms, and prevent TalkBack from
```

Comment:
Which other platforms? No platforms were mentioned before this line. 
Did you mean "To be consistent across all/more platforms."